packages/genesys-spark-components/.lintstagedrc.json

```diff
@@ -1,6 +1,6 @@
 {
   "*": "prettier --ignore-unknown --write",
-  "*.{css,html, scss}": "stylelint --fix",
+  "*.{css,html,scss}": "stylelint --fix",
```

Comment:
99% sure this typo is why these linting errors were not being picked up on commit.